app/assets/stylesheets/components/_header.scss

```diff
@@ -0,0 +1,52 @@
+.header {
+  @include header-gradient;
+  position: relative;
+  top: -30px;
+
+  padding: 15px 0;
+  background-color: $white-100;
+  box-shadow: 0 0 0 1px rgba(0, 0, 0, 0.01);
+
+  .container {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__left {
```

Comment:
@aetheryna Could it be that we make this class more meaningful?
So we can know what the thing that this contains for 🤔

Reply:
Whoops I forgot to remove all this, the header is just a glorified light bar on the application 😂 Updated in https://github.com/aetheryna/google-scraper-ruby/pull/35/commits/f347b499d2460bbd6f358239317f5953a1edd419